### ISSUE TITLE
Pointpillars pfe.onnx parsing error

### ISSUE THREAD

**Issue by hz3014** [ApolloAuto/apollo#12946]
Hi, I was trying to run your Pointpillars codes in my own environment and I noticed that all pretrained pfe.onnx(https://github.com/ApolloAuto/apollo/blob/4eb1312df1a7839a68ca51a88fe0e02df3f09179/modules/perception/production/data/perception/lidar/models/detection/point_pillars/pfe.onnx    AND   https://github.com/ApolloAuto/apollo/blob/3c89c6f02fc1b96f9a6f744283702d46ddd3a278/modules/perception/production/data/perception/lidar/models/detection/point_pillars/pfe.onnx ) couldn't be parsed by TensorRT properly. The parsing of rpn.onnx seems fine. 

It seems like there's problem with the Reshape. 
![pfeonnx](https://user-images.githubusercontent.com/35603799/98627090-54ad5900-234e-11eb-9d0f-b5cdb6a94b56.png)
Just wondering did you meet the problem and how did you solve it?

Error message:
Input filename:   /media/pfe.onnx
ONNX IR version:  0.0.4
Opset version:    9
Producer name:    pytorch
Producer version: 1.3
Domain:           
Model version:    0
Doc string:       
----------------------------------------------------------------
WARNING: onnx2trt_utils.cpp:220: Your ONNX model has been generated with INT64 weights, while TensorRT does not natively support INT64. Attempting to cast down to INT32.
[tcsetpgrp failed in terminal_inferior: Inappropriate ioctl for device]
[New Thread 0x7fff9e7fc700 (LWP 17115)]
ERROR: ../builder/myelin/codeGenerator.cpp (401) - Myelin Error in compileGraph: 64 (myelinProgramAnalysisError : Mistmatched number of elements in op (Unnamed Layer_ 46) [Shuffle]_reshape input tensor 49' shape [30000,1] and specified output shape [30000,60,1]
)
ERROR: ../builder/myelin/codeGenerator.cpp (401) - Myelin Error in compileGraph: 64 (myelinProgramAnalysisError : Mistmatched number of elements in op (Unnamed Layer_ 46) [Shuffle]_reshape input tensor 49' shape [30000,1] and specified output shape [30000,60,1]
)

### System information
- Linux Ubuntu 18.04:
- TensorRT 7.2.1:
- CUDA 11.1:

**Comment by jeroldchen** [comment]
@hz3014 PointPillars model is expected to run with TensorRT 7.0.0 and CUDA 10.2 which are Apollo 6.0's requirements. We haven't tested PointPillars model with TensorRT 7.2.1 nor CUDA 11.1.

**Comment by storypku** [comment]
@hz3014 Please use Apollo Dev container (or your own env with the same CUDA/cuDNN/TensorRT versions as @jeroldchen mentioned) to do your experiment first. Support of CUDA 11.x is still WIP.

**Comment by hz3014** [comment]
Thanks, just downgrade some environments and the onnx parsing is working now.
